web/src/components/storage/ProposalSettingsSection.jsx

```diff
@@ -167,55 +173,220 @@ file systems will be created on the selected device.");
 };
 
 /**
- * Allows to select LVM
+ * Form for configuring the system volume group.
+ * @component
+ *
+ * @param {object} props
+ * @param {string} props.id - Form ID.
+ * @param {ProposalSettings} props.settings - Settings used for calculating a proposal.
+ * @param {StorageDevice[]} [props.devices=[]] - Available storage devices.
+ * @param {onSubmitFn} [props.onSubmit=noop] - On submit callback.
+ * @param {onValidateFn} [props.onValidate=noop] - On validate callback.
+ *
+ * @callback onSubmitFn
+ * @param {string[]} devices - Name of the selected devices.
+ *
+ * @callback onValidateFn
+ * @param {boolean} valid
+ */
+const LVMSettingsForm = ({
+  id,
+  settings,
+  devices = [],
+  onSubmit: onSubmitProp = noop,
+  onValidate = noop
+}) => {
+  const [vgDevices, setVgDevices] = useState(settings.systemVGDevices);
+  const [isBootDeviceSelected, setIsBootDeviceSelected] = useState(settings.systemVGDevices.length === 0);
```

Comment:
NP: sometimes I think about if `is` is really needed. For example, what about

```suggestion
  const [bootDeviceSelected, setBootDeviceSelected] = useState(settings.systemVGDevices.length === 0);
```

Of course, `is` hints you about there should be a boolean value instead of _the boot device selected_... but I wanted to share my mess :stuck_out_tongue_closed_eyes:

Reply:
Yes, I think we should be consistent. IMHO, using `is` prefix for booleans is a good approach (similar to `?` sufix in ruby).